pkg/sql/compile/operator.go

```diff
@@ -609,6 +611,7 @@ func constructExternal(n *plan.Node, ctx context.Context, fileList []string) *ex
 		Es: &external.ExternalParam{
```

Comment:
insertNow if you need to execute remotely, please modify the corresponding proto file. thank you